molequeue/app/queues/queueuit.cpp

```diff
@@ -47,8 +47,8 @@
 #include <QtCore/QSettings>
 #include <QtCore/QDir>
 #include <QtXmlPatterns/QAbstractMessageHandler>
-#include <QtGui/QApplication>
```

Comment:
I would prefer to keep the module name in the include path, so using QtWidgets to replace QtGui would be preferable, otherwise this looks good.